client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -159,6 +159,30 @@ export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
   const okNextDisabled =
     (mode === ModalMode.Update && nextDisabled) || !currentItem;
 
+  const handleClick = async () => {
+    const allocated = draftStockOutLines?.reduce(
+      (accumulator, stockOutLine) => accumulator + stockOutLine.numberOfPacks,
+      0
+    );
```

Comment:
Maybe:
```
const noneAllocated =
      draftStockOutLines.filter(line => line.numberOfPacks > 0).length === 0;

if (noneAllocated) {
...
```

This is pretty nitpicky, yours works fine -- I just prefer a `filter` over a `reduce` when possible, they're usually easier to understand.

Up to you, but if you do leave it as is, then I'd remove the `?.` from `draftStockOutLines?.reduce` -- TypeScript is ensuring there will be something there (even if an empty array)

Reply:
Thank you for the prompt, I agree that is much more easy for me to read

Reply:
I'm going to nit pick a nit pick! 😄 ( though I agree with the comment about `reduce` - it works and is clear enough, but requires a bit of time to read and understand the intention )

Even the `filter` version requires a bit of re-reading to get the intention. I would use `every` like this:

```
  const noneAllocated = draftStockOutLines.every(line => line.numberOfPacks === 0);
```